README.md

```diff
@@ -60,7 +60,14 @@ ENV["MYSQL_DATABASE"] = "test_db"
 
 ## Development
 
-After cloning run `crystal deps` or `crystal deps update`.
+After cloning run:
+
+```
+git submodule init && git submodule update
```

Comment:
`git submodule update --init --recursive` is the most versatile command, that can be used in nearly any state.